qemu/tests/virtio_serial_hotplug_existed_port_pci.py

```diff
@@ -0,0 +1,40 @@
+from virttest import error_context
+from virttest import utils_test
+from virttest import qemu_monitor
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Add existed virtio serial port and serial bus
+    1) Start guest with virtio-serial-port and virtio-serial-pci
+    2) Hot-plug existed virtio-serial-port
+    3) Hot plug existed virtio-serial-pci
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    if params['os_type'] == 'windows':
+        utils_test.qemu.windrv_check_running_verifier(
+            vm.wait_for_login(), vm, test, 'vioser', 300)
```

Comment:
@michiboo Please move the `vm.wait_for_login()` out.
For linux guest,  without this step, it will start hotplug test during vm booting. It is not the expect behavior.